centrosome/kirsch.py

```diff
@@ -0,0 +1,21 @@
+import numpy
+import scipy.ndimage.filters
+
+
+def kirsch(image):
+    convolution_mask = [5, -3, -3, -3, -3, -3, 5, 5]
+
+    derivatives = numpy.zeros(image.shape)
+
```

Comment:
previously, the kernel was

    np.array(convolution_mask[:4] + [0] + convolution_mask[4:]).reshape(3,3)

this resulted in the following kernel on the first pass:

     5, -3, -3
    -3, 0, -3
    -3, 5,  5

instead of

    5, -3, -3
    5,  0, -3
    5, -3, -3

The way it's done is made a little more explicit by the ordering - you can see which elements go where (and it fixes the bug).